src/items.rs

```diff
@@ -2074,7 +2074,7 @@ fn rewrite_static(
             true,
         )
         .ok()
-        .and_then(|res| recover_comment_removed(res, static_parts.span, context))
+        .and_then(|res| Some(recover_comment_removed(res, static_parts.span, context)))
```

Comment:
Similar comment as before. It's probably nicer to call `map` so that we don't need to wrap the value in `Some`:

```suggestion
        .map(|res| recover_comment_removed(res, static_parts.span, context))
```